File: Day 3/3 Anagram String/solution.py
def areAnagram(str1, str2):
# Get lengths of both strings

    n1 = len(str1)
    n2 = len(str2)

    # If length of both strings is not same,
    # then they cannot be anagram
    if n1 != n2:
        return False

    # Sort both strings
    str1 = sorted(str1)
    str2 = sorted(str2)

    # Compare sorted strings
    for i in range(n1):
        if str1[i] != str2[i]:
            return False

    return True

File: Day 3/3 Anagram String/solution.cpp
#include <algorithm>
#include <string>

bool areAnagram(std::string str1, std::string str2)
{
    // Get lengths of both strings
    int n1 = str1.length();
    int n2 = str2.length();


    // If length of both strings is not same,
    // then they cannot be anagram
    if (n1 != n2)
        return false;

    // Sort both strings
    std::sort(str1.begin(), str1.end());
    std::sort(str2.begin(), str2.end());

    // Compare sorted strings
    for (int i = 0; i < n1; i++)
        if (str1[i] != str2[i])
            return false;

    return true;
}